app/src/components/billing/BillingContentTabs/Invoices/Invoices.tsx

```diff
@@ -0,0 +1,58 @@
+import { Card, VStack, Text, Icon, Link as ChakraLink } from "@chakra-ui/react";
+import { useInvoices } from "~/utils/hooks";
+import InvoicesTable from "./InvoicesTable";
+import NewTestInvoiceButton from "./NewTestInvoiceButton";
+import { FaTable } from "react-icons/fa";
+
+const Invoices = () => {
+  const query = useInvoices(10000);
+
+  const pendingCount = query.data?.invoices.filter((inv) => inv.status === "PENDING").length;
```

Comment:
It is a good point. I was thinking of doing it but moved the simplest way to avoid typing issues.
Once I have a `typedInvoice` helper it would be much easier to do.